gradle.properties

```diff
@@ -1,6 +1,6 @@
 # 1. Reduce Xmx after https://github.com/gradle/gradle-private/issues/4168 is resolved
 # 2. Enable html problem report for this build with -Dorg.gradle.internal.problems.report.enabled=true
-org.gradle.jvmargs=-Xmx3100m -XX:MaxMetaspaceSize=768m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx3500m -XX:MaxMetaspaceSize=768m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
🤔 This is concerning...

Reply:
Yeah, I am getting a lot of OOM problems with Gradleception tests with `-Xmx3100m` heap size, so I finally increased the memory - I plan to have a separate `[infra]` tagged commit for that

Reply:
Should we do these changes in separate PRs? I mean, if the need for more memory is not related to this PR, it shouldn't be part of it, no?

Reply:
Yes, I can also do it in another PR